internal/markdown/markdown.go

```diff
@@ -10,13 +12,29 @@ var (
 	underlineRe     = regexp.MustCompile(`(?ms)__(.*?)__`)
 	strikethroughRe = regexp.MustCompile(`(?ms)~~(.*?)~~`)
 	codeblockRe     = regexp.MustCompile("(?ms)`" + `([^` + "`" + `\n]+)` + "`")
+	emoteRe         = regexp.MustCompile(`<(:[a-zA-Z0-9]+:)[0-9]+>`)
+
+	cfg *config.Config
+	emoteColor string
 )
 
+func Init() {
+	var err error
+	cfg, err := config.Load()
+	if err != nil {
+		return
+	}
+	
+	emoteColor = cfg.Theme.MessagesText.EmoteColor
+}
+
 func Parse(input string) string {
```

Comment:
Just accept `emoteColor` as an argument to the `markdown.Parse` function instead of loading the configuration twice.

Reply:
Done, pls see last commit.